datacenterlight/locale/de/LC_MESSAGES/django.po

```diff
@@ -338,31 +346,23 @@ msgstr ""
 "\"https://stripe.com\" target=\"_blank\">Stripe</a> für die Bezahlung und "
 "speichern keine Informationen in unserer Datenbank."
 
-msgid ""
-"You are not making any payment yet. After submitting your card information, "
-"you will be taken to the Confirm Order Page."
+msgid "Last"
 msgstr ""
```

Comment:
Letztes
Letzte
Zurück (?)

Reply:
The context for this word is "Last 4" digits of a card. 
Last 4 = Letzten 4
Is this translation good? @MalcolmA

Reply:
Letzten 4 is okay. But you don't say it like that in german. You would rather say "Endziffern" but without the 4.